### DISCUSSION TITLE
Error Code: 2--0069 Skyline plugin has panicked!

### DISCUSSION THREAD

**Discussion by PrinceNestari** [Raytwo/ARCropolis#347] [Issues]
![KEcEO1-w](https://user-images.githubusercontent.com/82190743/169236262-6357362f-ae14-47de-b2e3-6b13aada6a5a.jpg)

**Comment by Raytwo** [comment]
Ok

**Reply by PrinceNestari** [reply]
Oh woops it's been awhile since I even made the post and I do apologize for not elaborating more on the issue. So my SD card has been formatted to FAT32 prior and Im still running into the issue whenever i get past the Switch logo

**Reply by Raytwo** [reply]
That’s not the problem. There is just no info, no context, nothing. Take a look at a solved discussion and see what they provide, then we’ll see. But at a glance, that’s an issue where I know the problem but can’t really solve it for the time being.
Try disabling your mods bit by bit until you can isolate which one causes the crash, that might give me a clue about the fix.
